sys/compat/linux/linux_file.c

```diff
@@ -610,8 +610,8 @@ linux_chmod(struct thread *td, struct linux_chmod_args *args)
 
 	LCONVPATHEXIST(td, args->path, &path);
 
-	error = kern_fchmodat(td, AT_FDCWD, UADDR2PTR(path), UIO_SYSSPACE,
```

Comment:
Is there a reason not to make `UADDR2PTR` do the right thing?

Reply:
It's not a macro that's defined anywhere (and it would be wrong since this isn't a user address). Given the commit date for this change (2018-01-29) this was probably and idea I abandoned and then didn't sweep up after due to the code not getting compiled and extreme sleep deprivation...